UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -807,6 +858,22 @@ public void ChangeFireStacks(float deltaValue)
 			}
 		}
 
+		public void ChangeBleedStacks(float deltaValue)
+		{
+			if (BleedStacks + deltaValue < 0)
```

Comment:
can't you simply add with a ``Mathf.Clamp`` to make sure it isn't less than zero and also more than the max?

Reply:
Alright, I'll add that to firestacks as well (since they also do this)

Reply:
ok, done